script.js

```diff
@@ -1,4 +1,111 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+var Counter = 0;
+var winningCounter = 0;
+//var myOutputvalue;
+var main = function (input, userID) {
+  Counter = Counter + 1;
+  console.log(
+    "hey " +
+      userID +
+      "! You have played " +
+      Counter +
+      "and you have been winning " +
+      winningCounter
+  );
+
+  var human = input;
+
+  if (inputValidation(human) != true) {
+    myOutputvalue = "please check your spelling";
+    return myOutputvalue;
```

Comment:
you can shorten it by using, 
if (input != "scissors" && input != "paper" && input != "stone") {
   myOutputvalue = "please check your spelling";
   return myOutputvalue;
  }